src/pass/storage_rewrite.cc

```diff
@@ -777,15 +786,24 @@ class StoragePlanRewriter : public IRMutator {
     return e;
   }
 
+  bool check_scope(const StorageEntry *e,
```

Comment:
since it is two liner, just fold it into the caller will make it easier to read

Reply:
yes, if you say so, i just hated the repeating lines